### DISCUSSION TITLE
Upgrading plugins in pdm

### DISCUSSION THREAD

**Discussion by jamestansx** [pdm-project/pdm#631] [Q&A] [answered]
Hi, 

How do I  upgrade the `pdm-venv` via `pdm plugin`. Currently, `pdm plugin` only has `add`, `list`, and `remove` options. I tried to `add` the plugin again but it won't work, the version's still remain the same. 

I think removing the plugin and reinstall would get me the latest version but I don't think this isn't a neat way to do.

**Comment by frostming** [comment, marked as answer]
add it again

pdm plugin add --pip-args='-U' pdm-venv

**Reply by jamestansx** [reply]
> pdm plugin add --pip-args='-U' pdm-venv

That works. Thanks
